src/cil.ml

```diff
@@ -1275,8 +1275,8 @@ let charType = TInt(IChar, [])
 let boolType = TInt(IBool, [])
 
 let charPtrType = TPtr(charType,[])
-let charConstPtrType = TPtr(TInt(IChar, [Attr("const", [])]),[])
-let stringLiteralType = ref charPtrType
+let charConstPtrType = TPtr(TInt(IChar, [Attr("const", []); Attr("pconst", [])]),[])
+let stringLiteralType = charPtrType
```

Comment:
Is it necessary to remove the ability to have this specified by the machine model?

Reply:
Because the type of string literals doesn't depend on the machine, but is fixed by the standard.